docs/reference/triggers.md

```diff
@@ -0,0 +1,42 @@
+# Triggers
+
+This is a list of the current options that are available to trigger an OpenFaaS Funtion
```

Comment:
OpenFaaS functions can be triggered easily by any kind of event. A small piece of code will convert from the event-source and trigger the function using the OpenFaaS Gateway API.

The most common use-case is HTTP which acts as a lingua franca between internet-connected systems.